.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the feature you'd like and what it will do**
+A clear and concise description of what you want to happen.
+
+
+**Why is this feature important?**
+A short description of the importance of this feature and what it will help you achieve and what it will help you achieve.
```

Comment:
There was a mistake on the template in the google doc. This line should read: 
A short description of the importance of this feature and what it will help you achieve.